### ISSUE TITLE
Loss function

### ISSUE THREAD

**Issue by huichen5796** [huichen5796/hiwi-2023#8]
- Loss
    - for pre-train on 5000 database: 
        - loss = cross entropy (CE)
    - for re-train on 10000 database: 
        - [ ] loss = CE(labeled data)
        - [ ] loss = CE(pseudo-labeled data)
        - [ ] loss = alpha * CE(labeled data) + beta * CE(pseudo-labeled data)
        - [ ] loss = alpha * CE(labeled data) + beta * CE(pseudo-labeled data with high Confidence) + gamma * CE(pseudo-labeled data with low Confidence) #5
